src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -31,14 +31,9 @@
 import { EuiIcon } from '@elastic/eui';
 import { i18n } from '@osd/i18n';
 import React from 'react';
-import {
-  ChromeNavLink,
-  ChromeRecentlyAccessedHistoryItem,
-  CoreStart,
-  WorkspaceAttribute,
-} from '../../..';
+import { ChromeNavLink, ChromeRecentlyAccessedHistoryItem, CoreStart } from '../../..';
 import { HttpStart } from '../../../http';
-import { InternalApplicationStart } from '../../../application/types';
+import { InternalApplicationStart } from '../../../application';
```

Comment:
Unnecessary change, please revert this line, it may be the same on function but may result in unnecessary code conflict when we merge our code back to OSD main/2.x .